.github/workflows/kfp-samples.yml

```diff
@@ -45,42 +45,17 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Free up disk space
-      run: ./.github/resources/scripts/free-disk-space.sh
+      run: |
+        sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+        docker system prune -af --volumes
+        docker image prune -af
```

Comment:
@akagami-harsh @juliusvonkohout, these commands should have been added to the `free-disk-space.sh` script. @akagami-harsh can you please send a follow-up PR?